docs/ds-implementation.md

```diff
@@ -17,10 +17,10 @@ Follow the new component directory structure:
 ```markdown
 src/
 └── components/
-    └── ComponentA/
-        ├── index.tsx
-        ├── ComponentA.stories.tsx
-        └── // Any other files as applicable (utils, child components, useHook, etc.)
+└── ComponentA/
+├── index.tsx
+├── ComponentA.stories.tsx
+└── // Any other files as applicable (utils, child components, useHook, etc.)
```

Comment:
@pettinarip Darnit. Every time I try to save the changes locally, it wants to fix the formatting. -_-

Reply:
Yea, I think is because it is coming as a new/modified file when you update the branch with dev (that is caught by prettier). Shouldn't happen in the upcoming PRs you make.